versioned_docs/version-1.15/using-dev-envs.mdx

```diff
@@ -180,7 +180,7 @@ Now save your changes and refresh the browser. Okteto CLI will automatically syn
 Server running on port 8080.
 ```
 
-Okteto's Development Environment immediately shows the changes you made on the deployed version of your application. You don't need to wait for a CI pipeline to complete before you can validate the changes you've made. This saves a ton of time.
+Okteto's Development Environment immediately picks up the changes you made on the deployed version of your application. You can see your changes by simply refreshing the browser. You don't need to wait for a CI pipeline to complete before you can validate the changes you've made. This saves a ton of time.
```

Comment:
As it was written before I thought that it could be implied that the browser would automatically refresh.